onnxruntime/core/providers/cpu/reduction/reduction_ops.h

```diff
@@ -102,9 +142,21 @@ inline bool reduce_isnan<int32_t>(int32_t) { return false; }
 template <>
 inline bool reduce_isnan<int64_t>(int64_t) { return false; }
 
+TensorOpCost ParallelReduceFastCost(int64_t n_row, int64_t n_col, int64_t element_size);
+
+class _ReduceAggregator {
+ public:
```

Comment:
nit: coding conventions don't allow using '_' in a class name. Suggest ReduceAggregatorBase.

Reply:
done